javascript/react/src/components/ManageWorkflow.types.ts

```diff
@@ -2,11 +2,15 @@
  * Copyright (c) Microsoft. All rights reserved.
  * Licensed under the MIT license. See LICENSE file in the project.
  */
-import type { TableContainer } from '@datashaper/tables'
 import type { Workflow } from '@datashaper/workflow'
 
 import type { TransformModalProps } from '../index.js'
 
+export enum WorkflowOrder {
```

Comment:
This could actually be a much more generic enum for better reusability. Something like `OrdinalDisplay`.

Reply:
Or the simpler `DisplayOrder` would be fine